### PR TITLE
Fix the section: video theme

### PR DESCRIPTION
**Pull request by gburgett**
Section: Video and Section: Video Highlight need to be responsive

refs #https://github.com/watermarkchurch/paper-signs/issues/61

### CODE REVIEW

File: wcc-contentful-app/app/views/sections/_video_highlight.html.erb [hunk 1]
```diff
@@ -10,8 +10,8 @@
       </div>
     <% end %>
   </div>
-  <div class="column medium-6">
-    <div class="video-responsive section-video-highlight__video">
+  <div class="section-video-highlight__video col-xs-12 col-md-6">
```
Comment (jpowell) [inline wcc-contentful-app/app/views/sections/_video_highlight.html.erb]: question - were we putting the `__` convention into the gem, or could we use the gem as an opportunity to use nesting convention (rewrite some of the WMR convention)?

Comment (gburgett) [reply wcc-contentful-app/app/views/sections/_video_highlight.html.erb]: I don't think we've discussed whether we will change conventions.  Would you like to put something on the calendar?

**Comment by gburgett** [comment]
@jpowell you approved the dependent PR in paper-signs but didn't approve this one.  Is it still pending discussion on the conventions, or is that going to be separate?

**Comment by jpowell** [comment]
should be separate i suppose since we already have BEM styles in the gem at this point

**Pull request merged**